tests/returntypes/test_parameter_shift_shot_vec_new.py

```diff
@@ -2497,7 +2497,9 @@ def test_jax(self, tol, broadcast):
 """
 
 
-@pytest.mark.parametrize("shot_vec", [(100, 1, 10), (1, 1, 10), (*[1] * 2, 10), (1, 1, 1)])
+@pytest.mark.parametrize(
+    "shot_vec", [(100, 1, 10), (1, 1, 10), ((1, 2), 10), (10, (1, 2)), (1, 1, 1)]
```

Comment:
I don't think (1,2) will work because it is not processed as a list of ShotTuple.

Reply:
Great catch! Thank you.